core/jobs/batch_jobs/manual_voice_artist_name_job.py

```diff
@@ -253,33 +237,41 @@ def get_exploration_voice_artists_link_model(
             new_snapshot_id = exploration_model.id + '-' + str(version)
             old_snapshot_id = exploration_model.id + '-' + str(version - 1)
 
-            logging.info(
-                'Current iteration for snapshots: %s and %s\n' % (
+            debug_logs += (
+                'Iteration for snapshots: %s and %s\n' % (
                     old_snapshot_id, new_snapshot_id))
-            logging.info('Thread ID: %s\n' % threading.get_native_id())
+            logging.info('Iteration for snapshots: %s and %s\n' % (
+                old_snapshot_id, new_snapshot_id))
 
             if old_snapshot_id not in snapshot_models_dict:
+                logging.info(
+                    '%s not present in snapshot model dict.' % old_snapshot_id)
                 continue
             if new_snapshot_id not in snapshot_models_dict:
+                logging.info(
+                    '%s not present in snapshot model dict.' % new_snapshot_id)
                 continue
             if new_snapshot_id not in metadata_models_dict:
+                logging.info(
+                    '%s not present in metadata model dict.' % new_snapshot_id)
                 continue
 
             new_snapshot_model = snapshot_models_dict[new_snapshot_id]
             old_snapshot_model = snapshot_models_dict[old_snapshot_id]
 
-            # If the commit does not contain voiceover changes, then we should
-            # skip the snapshot model.
-            if not cls.is_voiceover_changes_made(
```

Comment:
Not required, the code below (in the try block) is capable of doing the same thing. Hence this is redundant.